g3w-admin/client/views.py

```diff
@@ -169,12 +169,23 @@ def get_context_data(self, **kwargs):
             raise Http404('No project type and/or project id present in group')
 
         # page title
-
         contextData['page_title'] = '{} | {}'.format(
             getattr(settings, 'G3WSUITE_CUSTOM_TITLE', 'g3w - client'),
             self.project.title_ur if self.project.title_ur else self.project.title
         )
 
+        # server side styles (inline css)
+        contextData['CLIENT_CUSTOM_CSS'] = ''
+
+        # Conditionally show sidebar items
+        elements_to_hide = []
+
+        if not getattr(self.project, 'show_metadata_section', True):
+            elements_to_hide.append('#metadata')
+
+        if elements_to_hide:
+            contextData['CLIENT_CUSTOM_CSS'] += f".sidebar-menu > li:is({', '.join(elements_to_hide)}) {{ display: none; }}"
+
```

Comment:
Ref: https://github.com/g3w-suite/g3w-admin/issues/1099#issuecomment-2853438545 for any other one:


```py
elements_to_hide = ['#metadata', '#spatialbookmarks', '#print'] # here your django options.. 😃
CLIENT_CUSTOM_CSS = f".sidebar-menu > li:is({', '.join(elements_to_hide)})"
```